app/views/apis/index.html.erb

```diff
@@ -1 +1,11 @@
 API Index Page
+
+<div class="apis">
+  <% @apis.each do |api| %>
+    <li>
```

Comment:
<li> elements semantically belong inside of a <ul> or <ol> tag. Using semantic HTML is important for search engines and accessibility devices like screen readers.